popup.html

```diff
@@ -29,4 +29,4 @@
     <div id="status"></div>
     <img id="image-result" hidden="">
```

Comment:
I will remove it for cleanliness sake, but all of the code in popup.html is being rewritten anyway (no point in having the redundant search happening in the popup). Will be replaced by dropdown with 2 things -- one a pause bottom, two a link to a webpage with you saved movies. Fyi, that's v2 (local storage). I texted you this but v3 I realized is persistent storage with a databse...and to use this tool as the way people save their movies.